### PR TITLE
Blobs

### PR DESCRIPTION
**Pull request by axelbacklund**
Adding blob option to spacing elements!

### CODE REVIEW

File: src/utils/BlobDirection.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+export type BlobDirection = 'none' | 'up' | 'down'
+export interface BaseBlobProps {
```
Comment (palmenhq) [inline src/utils/BlobDirection.ts]: i loled, this name is amazing 😂

File: src/utils/BlobDirection.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    top: 'auto',
+    bottom: 0,
+  },
+  down: {
+    top: 0,
+    bottom: 'auto',
+  },
+}
+
+const upBlobPathData: string =
```
Comment (palmenhq) [inline src/utils/BlobDirection.ts]: not that it's bad to have it, but i think TS can infer the `string` type :)

Comment (axelbacklund) [reply src/utils/BlobDirection.ts]: True!

**Comment by axelbacklund** [comment]
Thanks!! 😃Just thinking, if we're removing the inherent spacing of blocks per Oscar's PR, maybe it will be enough for now using the spacer blocks?

**Comment by axelbacklund** [comment]
Checked with the group, should be fine with just spacer blocks 👍

**Pull request merged**